gp-templates/translations.php

```diff
@@ -165,9 +174,20 @@
 		 *
 		 * This action is inside a DL element.
 		 *
+		 * @deprecated 2.1.0 Call gp_translation_set_sort_form instead
```

Comment:
I prefer keeping it in the template so it's obvious to developers that the old call is deprecated and will eventually be removed.

Placing it in a call from another `do_action()` seems too much like hiding it to me :wink:.